server/datastore/mysql/teams_test.go

```diff
@@ -424,3 +426,48 @@ func testTeamsDeleteIntegrationsFromTeams(t *testing.T, ds *Datastore) {
 	require.NoError(t, err)
 	assertIntgURLs([]string{urld}, []string{urle, urlf}, []string{urle, urlf})
 }
+
+func testTeamsFeatures(t *testing.T, ds *Datastore) {
+	defaultFeatures := fleet.Features{}
+	defaultFeatures.ApplyDefaultsForNewInstalls()
+	ctx := context.Background()
+
+	t.Run("NULL config in the database", func(t *testing.T) {
+		team, err := ds.NewTeam(ctx, &fleet.Team{Name: "team_null_config"})
+		require.NoError(t, err)
+		ExecAdhocSQL(t, ds, func(tx sqlx.ExtContext) error {
+			_, err = tx.ExecContext(
+				ctx,
+				"UPDATE teams SET config = NULL WHERE id = ?",
```

Comment:
Do we have (or need) a test with not-`NULL` `config` but without `features` key?